highfidelity/hifi
# Fix several aspects of the "disconnected" dialog

Changes:

 1. On first load, wait longer to show the disconnected dialog
 1. If trying to go to a place that requires login, don't show the can't connect dialog
 1. Make the can't connect dialog go away if you get connected to the domain

Note: The disconnected dialog is the one that says "No Connection" at the top

To test:
 1. Go to your home domain, ensure that the disconnected dialog does not show
 1. Shut your local server down, ensure that the disconnected dialog comes up
 1. Manually close the dialog, start the server back up
 1. Shut your local server down, ensure that the disconnected dialog comes up
 1. DO NOT close the dialog, start the server back up, ensure that the dialog goes away automatically
 1. Shut your local server down, ensure that the disconnected dialog comes up
 1. Restart interface, ensure that the disconnected dialog comes up after more than 10 seconds
 1. Close the dialog
 1. Ensure you are logged out. Go to `sport`
 1. Ensure that the "login" dialog comes up but that the disconnected dialog does __not__
 1. Go to a domain running a different protocol version (`welcome`, for instance)
 1. Ensure the packet version mismatch pops up, but the disconnected dialog does __not__
